builder-support/helpers/install_quiche.sh

```diff
@@ -42,6 +42,15 @@ sed -i 's/ffi = \["dep:cdylib-link-lines"\]/ffi = \[\]/' quiche/Cargo.toml
 sed -i 's,cdylib_link_lines::metabuild();,//cdylib_link_lines::metabuild();,' quiche/src/build.rs
 RUST_BACKTRACE=1 cargo build --release --no-default-features --features ffi,boringssl-boring-crate --package quiche
 
+# While we tried to get rid of the SONAME in libquiche.so, on debian trixie's
+# packaged rustc puts it in anyway.
+# See (https://sources.debian.org/patches/rustc/1.85.0%2Bdfsg2-3/behaviour/d-rustc-add-soname.patch/).
+# So if it is present, patch it to the correct name.
+if objdump -p target/release/libquiche.${SOEXT} | fgrep -q SONAME
```

Comment:
That looks good, but perhaps it would be nice not to fail if `objdump` is not present? I'm OK with failing if `patchelf` is not present since we actually need it if a SONAME is present.

Reply:
If `objdump` is not present and we skip the test we might be creating an invalid lib. That's also not nice...

Reply:
I checked `build-and-test-all` and did package builds on `el-8` and `trixie` with `builder/build.sh`: all have `objdump` available.

Reply:
OK, then.